meerkat_frontend/src/js/technical/misc.js

```diff
@@ -1,3 +1,35 @@
+
+//This Code will handle the spinner all over the site ...
```

Comment:
Tiny thing thats not that important... Probably don't need to finish every comment with an elipsis "...". But it probably is a good thing to put a space ater the `//` e.g. 
```// This code will show the throbber for AJAX requests all over the site.```